packages/driver/cypress/e2e/e2e/origin/validation.cy.ts

```diff
@@ -1,6 +1,7 @@
 describe('cy.origin', { browser: '!webkit' }, () => {
   describe('successes', () => {
     it('succeeds on a localhost domain name', () => {
+      cy.visit('')
```

Comment:
added explicit visits to each test to allow for re-running the tests via cypress open

Reply:
This is to reset top

Reply:
Couldn't it be in a `beforeEach` with tests that don't need it in a separate suite?

Reply:
yeah, that would work too.

Reply:
I broke these into separate suites.